src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const order of actions) {
+    switch (order.type) {
+      case 'addProperties':
+        Object.assign(state, order.extraData);
+
+        break;
+
+      case 'removeProperties':
```

Comment:
good job working on this task, just to make sure , is it good idea to leave a blank line after and before the break statement?